datasets/wit/README.md

```diff
@@ -0,0 +1,236 @@
+## Table of Contents
+- [Table of Contents](#table-of-contents)
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks and Leaderboards](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+  - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:** https://github.com/google-research-datasets/wit
+- **Repository:** https://github.com/google-research-datasets/wit
+- **Paper:** https://arxiv.org/abs/2103.01913
+- **Leaderboard:** https://www.kaggle.com/c/wikipedia-image-caption
+- **Point of Contact:** krishnaps@google.com
+
+### Dataset Summary
+
+Wikipedia-based Image Text (WIT) Dataset is a large multimodal multilingual dataset. WIT is composed of a curated set of 37.6 million entity rich image-text examples with 11.5 million unique images across 108 Wikipedia languages. Its size enables WIT to be used as a pretraining dataset for multimodal machine learning models.
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+The dataset contains examples from all Wikipedia languages, with the following stats:
+
+Image-Text   | # Lang | Uniq. Images  | # Lang
+------------ | ------ | ------------- | ------
+total > 1M   | 9      | images > 1M   | 6
+total > 500K | 10     | images > 500K | 12
+total > 100K | 36     | images > 100K | 35
+total > 50K  | 15     | images > 50K  | 17
+total > 14K  | 38     | images > 13K  | 38
+
+## Dataset Structure
+
+### Data Instances
+
+```
+
+```
+### Data Fields
+
+- `language`
+- `page_url`
+- `image_url`
+- `page_title`
+- `section_title`
+- `hierarchical_section_title`
+- `caption_reference_description`
+- `caption_attribution_description`
+- `caption_alt_text_description`
+- `mime_type`
+- `original_height`
+- `original_width`
+- `is_main_image`
+- `attribution_passes_lang_id`
+- `page_changed_recently`
+- `context_page_description`
+- `context_section_description`
+
+Details on the field content can be found directly in the [paper, figure 5 and table 12.](https://arxiv.org/abs/2103.01913)
+
+### Data Splits
+
+## Dataset Creation
+
+### Curation Rationale
+
+[More Information Needed]
```

Comment:
Feel free to copy and paste this [section](https://github.com/google-research-datasets/wit#motivation) the GH repo here